openshift/topics/tutorials.adoc

```diff
@@ -740,48 +778,54 @@ $ systemctl restart atomic-openshift-master
 
 ==== Logging in to OpenShift
 
-Navigate to the OpenShift web console, which in this example is _https://openshift.example.com:8443/console_. The OpenShift login page now has the option to use either *htpasswd_auth* or *rh-sso*. The former is still available because it is present in the */etc/origin/master/master-config.yaml*.
+.Procedure
+
+. Navigate to the OpenShift web console, which in this example is _https://openshift.example.com:8443/console_.
++
+The OpenShift login page now has the option to use either *htpasswd_auth* or *rh-sso*. The former is still available because it is present in the */etc/origin/master/master-config.yaml*.
```

Comment:
Maybe `The OpenShift login page now offers the options to login either using *htpasswd_auth* or *rh-sso* identity providers?`